core/src/main/java/net/opentsdb/query/execution/serdes/JsonV3QuerySerdes.java

```diff
@@ -233,21 +236,16 @@ public Object call(final ArrayList<TimeSeriesStringId> ids)
           } else {
             for (final TimeSeries series :
               series != null ? series : result.timeSeries()) {
+              Map<String, String> additionalProps = new HashMap<>();
```

Comment:
So instead of props and the iterator type check, do `series.types().contains(StatusType.TYPE);` and it'll be much cleaner.